scss/_variables.scss

```diff
@@ -565,6 +565,7 @@ $form-switch-width:               2em !default;
 $form-switch-height:              $form-check-input-width !default;
 $form-switch-padding-left:        $form-switch-width + .5em !default;
 $form-switch-bg-image:            url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'><circle r='3' fill='#{$form-switch-color}'/></svg>") !default;
+$form-switch-bg-size:             calc(#{$form-switch-height} - 2px) !default;
```

Comment:
Maybe switch to `contain` here?

Reply:
`contain`?

Reply:
`background-size: contain;`, feels a little more flexible.